CMakeLists.txt

```diff
@@ -20,6 +20,29 @@ endif ()
 project(remill)
 cmake_minimum_required(VERSION 3.2)
 
+if(EMSCRIPTEN)
+  set(CMAKE_BUILD_TYPE Release)
```

Comment:
Can this be moved at the top of the cmake/settings.cmake file? If this new feature does not support other build configurations (such as Debug), we could add a warning message and overwrite the setting.

Example:
```
if(REMILL_EMSCRIPTEN)
  ..
  if(NOT "${CMAKE_BUILD_TYPE}" STREQUAL "Release")
    message(WARNING "remill: The REMILL_EMSCRIPTEN overwrites the build type to Release")
    set(CMAKE_BUILD_TYPE "Release" CACHE STRING "Build type" FORCE)
  endif()
endif()
```

Reply:
Yeah I love this idea!

Reply:
Has this been resolved?